Zilon.Core/Zilon.Bot.Players.DevelopmentTests/BotActorTaskSourceTests.cs

```diff
@@ -85,6 +85,12 @@ private static void PrintPersonBacklog(IPerson humanPerson)
                         break;
                 }
             }
+
+            Console.WriteLine("Start attributes:");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5484619)